### PR TITLE
Improve password manager recommendations

### PR DESCRIPTION
**Pull request by cypherpunk-institute**
Get recommendations inline with reputable and widely-peer-reviewed opinion of /r/privacytoolsio community (specifically https://www.privacytools.io/software/passwords/).

### CODE REVIEW

**Comment by krakensecuritylabs** [comment]
Thanks! 

Keepass and Bitwarden are great suggestions and we have added them to https://github.com/krakensecuritylabs/canisecure/commit/ee079aeb21aa4ab1b5fbdfdf892d4ad13294400f

PrivacyTools is a great resource but is aimed at users with a slightly different threat model than who we are targeting with canisecure. We understand the risks of using a cloud-based PW managers but we think the risk is worth the adoption rate among less tech-savvy users.

**Comment by cypherpunk-institute** [comment]
Cryptocurrency users do store backup seed words of their mobile wallets, exchange credentials, TOTP initial shared secrets, and other critical secrets in their password manager.

So I guess this is even more relevant in the cryptocurrency space than in the privacy space.

The problem with "remote" password managers is they only offer a single layer of security - the encryption alone (SPOF). The "local" PMs offer a second layer: data being local.

Please see this resource for real world vulns in password managers:
https://password-managers.bestreviews.net/faq/which-password-managers-have-been-hacked/

**Comment by cypherpunk-institute** [comment]
PS Sorry for being pain in the ass - I am considering recommending your canisecure.com broadly (instead of answering the same questions all over again). What you have is an excellent start; I just  need to patch a few things ;)

**Comment by krakensecuritylabs** [comment]
We appreciate the conversation and that you like this resource. 

In a vacuum, you're right that local managers are more secure than remote managers. In the real world, the best password manager is the one that you use all the time, on every device. We don't think most folks are using any password manager right now and aren't going to invest the energy to get Keepass correctly configured, sync'd to their phone and backed up. 

The resource you linked is good, but also ends with saying that people should continue to use PW managers (and doesn't differentiate between local/remote). To our knowledge, none of those vulnerabilities led to wide-scale compromises of user passwords. However, we see a huge amount of credential stuffing per day (including at kraken.com) because people are simply reusing passwords.

TLDR We don't want perfect to be the enemy of good
